streamly-shell.cabal

```diff
@@ -66,4 +66,5 @@ library
     -Wredundant-constraints
     -Wnoncanonical-monad-instances
   build-depends:
-      base              >= 4.8   && < 5
+        base >= 4.8 && < 5
+      , streamly >= 0.8.0
```

Comment:
Would this compile for `0.8.1`?

Reply:
I forgot to change the upper bound. It should compile with any version that exports the `SerialT` type.